app/models/GenericListItem.scala

```diff
@@ -0,0 +1,31 @@
+/*
```

Comment:
Is this ReferenceDataList renamed/updated?

Reply:
No, there is still `ReferenceDataList`, this is just an intermediate object that enriches each items with the extra data needed for indexing and querying, since we're going with a single collection for items